src/Microsoft.Diagnostics.EventFlow.Inputs.EventSource/EventDataExtensions.cs

```diff
@@ -29,6 +30,11 @@ public static EventData ToEventData(this EventWrittenEventArgs eventSourceEvent)
             eventPayload["EventId"] = eventSourceEvent.EventId;
             eventPayload["EventName"] = eventSourceEvent.EventName;
             eventPayload["ActivityID"] = ActivityPathString(eventSourceEvent.ActivityId);
+            var eventContext = ContextResolver.GetRequestContext<object>();
+            if (eventContext != null)
+            {
+                eventPayload["EventContext"] = eventContext;
```

Comment:
1. Oh, then this API is misleading. Let me fix this API as a part of Correlation release process, and stick with `EventContext` here.